### PR TITLE
Try enabling Docker CD with Now GitHub integration

### PR DESCRIPTION
**Pull request by mxstbr**
ref https://zeit.co/docs/static-deployments/builds/building-with-now

### CODE REVIEW

**Comment by quietshu** [comment]
Hi @mxstbr I'm getting this error when I do `yarn export` locally 😅:

```
> exporting path: /appearances
{ Invariant Violation: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
```

And also I saw you copied `now.json` as static file here: 
https://github.com/mxstbr/mxstbr.com/blob/master/next.config.js#L9
It'll cause an error and I guess it's been ignored somewhere. 
Maybe we need to remove it from `staticFilesToCopy`.

**Comment by mxstbr** [comment]
Interesting, I get a different error:

```
yarn run v1.7.0
$ next build && next export -o /public
[17:39:19] Compiling client
[17:39:19] Compiling server
> Using external babel configuration
> Location: "/Users/mxstbr/projects/github/mxstbr/mxstbr.com/.babelrc"
[17:39:21] Compiled server in 2s
[17:39:23] Compiled client in 4s
> using build directory: /Users/mxstbr/projects/github/mxstbr/mxstbr.com/.next
{ Error: EACCES: permission denied, mkdir '/public' errono: -13, code: 'EACCES', syscall: 'mkdir', path: '/public' }
```

**Comment by mxstbr** [comment]
Aha found the cause for that error, fixed in the latest commit!

> And also I saw you copied now.json as static file here:

Hmm you're right, that might not be necessary anymore! Let me see what happens if I remove that, maybe that'll fix it 🤞

**Comment by mxstbr** [comment]
Oh dang now I get the same error you get, hmm very strange. Why can't it `next export` while CD'ing?

**Comment by mxstbr** [comment]
Not sure how to resolve this, not a lot of info to go on:

```
10/15 05:45 PM (2m)
{ Invariant Violation: Minified React error #130; visit https://reactjs.org/docs/error-decoder.html?invariant=130&args[]=undefined&args[]= for the full message or use the non-minified dev environment for full errors and additional helpful warnings. 
10/15 05:45 PM (2m)
    at ba (/usr/src/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:10:312)
10/15 05:45 PM (2m)
    at t (/usr/src/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:11:166)
10/15 05:45 PM (2m)
    at a.render (/usr/src/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:37:16)
10/15 05:45 PM (2m)
    at a.read (/usr/src/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:34:288)
10/15 05:45 PM (2m)
    at renderToString (/usr/src/node_modules/react-dom/cjs/react-dom-server.node.production.min.js:45:6)
10/15 05:45 PM (2m)
    at renderPage (/usr/src/node_modules/next/dist/server/render.js:319:26)
10/15 05:45 PM (2m)
    at Function.getInitialProps (/usr/src/.next/server/static/ABxgsEp3UPW1SH40nzU_Z/pages/_document.js:199:18)
10/15 05:45 PM (2m)
    at _callee$ (/usr/src/node_modules/next/dist/lib/utils.js:86:30)
10/15 05:45 PM (2m)
    at tryCatch (/usr/src/node_modules/regenerator-runtime/runtime.js:62:40)
10/15 05:45 PM (2m)
    at Generator.invoke [as _invoke] (/usr/src/node_modules/regenerator-runtime/runtime.js:288:22) name: 'Invariant Violation', framesToPop: 1 }
```

**Comment by quietshu** [comment]
Yeah, seems it's not a problem of Docker CD since I can repro locally. 😢

**Comment by mxstbr** [comment]
Wait but if you can repro locally, how come it works on my machine? Wut?

**Comment by mxstbr** [comment]
Are you running on master or this branch?

**Comment by quietshu** [comment]
On master, it's weird...

**Comment by mxstbr** [comment]
Can you try to run the site in dev mode? `yarn dev` Does that also throw the error?

**Comment by quietshu** [comment]
Yeah I can see the same error in dev mode:
image

**Comment by mxstbr** [comment]
🤔 why does that happen for you but not for me?

File: package.json [hunk 1]
```diff
@@ -7,7 +7,7 @@
     "dev": "next",
     "build": "next build",
     "start": "next start",
-    "export": "next build && next export",
+    "export": "next build && next export -o public",
```
Comment (lucleray) [inline package.json]: If you want to use this to export for now, it should be this (notice the `/` before public) :
```suggestion
    "export": "next build && next export -o /public",
```

But it's probably better to keep it like this :
```suggestion
    "export": "next export",
```

And then, in the dockerfile, use yarn export -o /public

**Pull request closed without merging**